pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -280,6 +280,16 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (result re
 		return reconcile.Result{}, errors.Wrap(err, "failed to create provisioner")
 	}
 
+	provStatus, err := prov.IsReady()
+	if err != nil {
+		return reconcile.Result{}, errors.Wrap(err,
+			fmt.Sprintf("failed to check services availability"))
```

Comment:
This could just be an inline string, since there are no additional variables in the Sprintf(). That can be fixed in a follow-up PR unless there are other changes needed for this one.

Reply:
Done